SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -12,11 +12,18 @@ Volume {
 	}
 
 	init {
-		if(server.serverRunning) { this.sendSynthDef };
+		// execute immediately if we're already past server tree functions
+		if(server.serverRunning) {
+			this.sendSynthDef;
+			this.updateSynth;
+		};
 
 		initFunc = {
 			ampSynth = nil;
-			this.sendSynthDef
+			this.sendSynthDef;
+
+			// only create synth now if it won't be created by ServerTree
+			if (persist.not) { this.updateSynth };
```

Comment:
Since you've made better use of `persist` with these changes, would be a good idea to make that variable `True` by default? I would be easier to read this class file and understand it's logic if this was the case.

Reply:
That kind of API change is out of scope of this PR and I have no opinion on whether that should be the case.

Reply:
That's a good point.